tests/test_dice_interface/test_dice_pytorch.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+import dice_ml
+from dice_ml.utils import helpers
+import numpy as np
+
+torch = pytest.importorskip("torch")
+
+@pytest.fixture
+def pyt_exp_object():
```

Comment:
It is not straightforward to put pyt_exp_object in conftest.py or have a different scope for two reasons:
1. It is not trivial to pass parameters from one fixture to a parent one (https://github.com/pytest-dev/pytest/issues/1694), and both `_initiate_exp_object` and `pyt_exp_object` are fixtures.
2.  `_initiate_exp_object` needs to be a fixture and had to use it with `autouse=True` to avoid calling the fixture and initiating the explainer object repeatedly.
Perhaps, there's a way to address this, so I'm open to suggestions.